client/src/components/spoolIcon.css

```diff
@@ -3,7 +3,6 @@
   width: 1.5em;
   height: 2.5em;
   gap: 2px;
-  margin: 0 0.5em;
```

Comment:
This doesn't look so good in the spool list, the text is butted up against the color block. Why did you remove this?

Reply:
Just an UI/UX improvement.

During development, I saw, that the color is not aligned to the same spot as other records. I thought this was not intended.

![image](https://github.com/user-attachments/assets/893a01c4-595f-4f40-9c08-f0d3d425ab4b)

If this was your intention, I will revert the change 😊

Reply:
Oh okay, I see the issue! Will fix that